bin/hledger-git

```diff
@@ -74,8 +74,11 @@ record() {
     ensure_git_repo
     cd "$DIR"
     for F in $FILES; do $GIT add -f "$F"; done
-    MSG=${1:-$(date +'%Y-%m-%d %H:%M:%S %Z')}; shift
-    $GIT record -m "$MSG" "$@" -- "$FILES"
+    MSG=${1:-$(date +'%Y-%m-%d %H:%M:%S %Z')}
+    if [ $# -ge 1 ]; then
+        shift
+    fi
+    $GIT commit -m "$MSG" "$@" || $GIT reset
```

Comment:
minor: `"$@"` is only possible if a custom message is given. However to change this would change the previous/existing behavior since it was like that.